docker/vars.local.e2e.sh

```diff
@@ -4,7 +4,8 @@ export GP2GP_SERVER_PORT="8080"
 export GP2GP_AMQP_BROKERS="amqp://activemq:5672"
 export GP2GP_MONGO_URI="mongodb://mongodb:27017"
 export GP2GP_MONGO_DATABASE_NAME="gp2gp"
-export GP2GP_MHS_OUTBOUND_URL="http://mock-mhs-adaptor:8081/mock-mhs-endpoint"
+export GP2GP_MHS_MOCK_BASE_URL="http://mock-mhs-adaptor:8081"
+export GP2GP_MHS_OUTBOUND_URL="`echo $GP2GP_MHS_MOCK_BASE_URL`/mock-mhs-endpoint"
```

Comment:
that looks a bit unusual - i.e. isn't the interpolation in double quotes working? export GP2GP_MHS_OUTBOUND_URL="$GP2GP_MHS_MOCK_BASE_URL/mock-mhs-endpoint"?

Reply:
indeed, it works as you wrote
changing